.github/workflows/notify-slack-on-label.yml

```
@@ -0,0 +1,22 @@
+name: Notify Slack on Needs Testing Label
+
+on:
+    issues:
+        types: [labeled]
+    pull_request:
+        types: [labeled]
+
+jobs:
+    notify:
+        if: ${{ github.event.label.name == 'Needs Testing' }}
+        runs-on: ubuntu-latest
+        steps:
+            - name: Send Slack Notification
+              uses: slackapi/slack-github-action@v2.1.1
+              with:
+                  webhook: ${{ secrets.SLACK_WEBHOOK_URL }}
```

```suggestion
                  webhook: ${{ secrets.SLACK_NEEDS_TESTING_LABEL_ADDED_WEBHOOK_URL }}
```

It's reasonable to anticipate additional Slack notifications to be needed in the future, so let's make the name of this secret more specific.